packages/keystone/fields/Json/Implementation.js

```diff
@@ -58,6 +64,29 @@ class JsonImplementation extends Implementation {
         return this.extendGraphQLTypes
     }
 
+    gqlAuxFieldResolvers ({ schemaName }) {
```

Comment:
It looks like a hardcode! Package should know nothing about the apps.

Reply:
Refactored at https://github.com/open-condo-software/condo/pull/2936